packages/integration-tests/suites/tracing/browsertracing/meta/test.ts

```diff
@@ -21,17 +21,19 @@ sentryTest(
   },
 );
 
-// TODO this we can't really test until we actually propagate sentry- entries in baggage
-// skipping for now but this must be adjusted later on
-sentryTest.skip(
-  'should pick up `baggage` <meta> tag and propagate the content in transaction',
+sentryTest(
+  'should pick up `baggage` <meta> tag, propagate the content in transaction and not add own data',
   async ({ getLocalTestPath, page }) => {
     const url = await getLocalTestPath({ testDir: __dirname });
 
     const envHeader = await getFirstSentryEnvelopeRequest<EventEnvelopeHeaders>(page, url, envelopeHeaderRequestParser);
 
     expect(envHeader.trace).toBeDefined();
-    expect(envHeader.trace).toEqual('{version:2.1.12}');
+    expect(envHeader.trace).toMatchObject({
+      public_key: 'public',
+      trace_id: expect.any(String),
+      release: '2.1.12',
+    });
```

Comment:
This is a nit, but will there be anything else in the `trace` header besides these three values? If not, I think continuing to use `toEqual` is a better choice, because it says more clearly what we mean.

(Forgive me if you already know this, but in spite of the name, `toMatchObject` doesn't actually mean the objects are an exact match, but rather that one is a subset of the other, more like `objectContaining` (though not quite the same, see https://stackoverflow.com/a/46876473). For that reason, I try to stay away from it and just use either `toEqual()` or `toEqual(expect.objectContaining())`, because they actually do what they say.)

Reply:
Good call, adjusted it in 212605faaf1dee63eb80590fda4e6a32139e0392